vector/src/fdroid/java/im/vector/riotx/fdroid/receiver/AlarmSyncBroadcastReceiver.kt

```diff
@@ -48,7 +48,7 @@ class AlarmSyncBroadcastReceiver : BroadcastReceiver() {
             }
         }
 
-        val sessionId = intent.getStringExtra(SyncService.EXTRA_SESSION_ID)
+        val sessionId = intent.getStringExtra(SyncService.EXTRA_SESSION_ID) ?: return
```

Comment:
If we return, the alarm will never be launched again?

Reply:
I think this was a temporary crash, a scheduled alarm with a previous version of RiotX. Not sure though... The rageshake was quite old

Reply:
It had been fixed 2 months ago